### ISSUE TITLE
ensime-sbt 2.0.1 - dependence conflict (compat) with sbt-revolver

### ISSUE THREAD

**Issue by strobe** [ensime/ensime-sbt#369]
I found dependence conflict with ensime-sbt 2.0.1 and sbt-revolver 0.9.0.
`package sbt contains object and package with same name: compat`

(also seems ﻿ensime-sbt 1.12.15 doesn't has this issue)

### Steps to reproduce

Project that may help to reproduce https://github.com/strobe/sbt-ensime2.0.1-revolver-compat-issue

1.  add addSbtPlugin("org.ensime" % "sbt-ensime" % "2.0.1") to ~/.sbt/0.13/plugins/plugins.sbt
2. add addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.0") to ~/project/plugins.sbt 
3. start sbt at project folder

### Actual behavior
sbt failed with message: 
```
[info] Loading global plugins from /Users/evgeniy/.sbt/0.13/plugins/project
[info] Loading global plugins from /Users/evgeniy/.sbt/0.13/plugins
[info] Loading project definition from /Users/evgeniy/tmp/sbt-ensime2.0.1-revolver-compat-issue/project
[info] Compiling 1 Scala source to /Users/evgeniy/tmp/sbt-ensime2.0.1-revolver-compat-issue/project/target/scala-2.10/sbt-0.13/classes...
[error] /Users/evgeniy/tmp/sbt-ensime2.0.1-revolver-compat-issue/project/Dependencies.scala:1: package sbt contains object and package with same name: compat
[error] one of them needs to be removed from classpath
[error] import sbt._
[error]        ^
[error] /Users/evgeniy/tmp/sbt-ensime2.0.1-revolver-compat-issue/project/Dependencies.scala:4: value %% is not a member of String
[error]   lazy val scalaTest = "org.scalatest" %% "scalatest" % "3.0.3"
[error]                                        ^
[error] two errors found
[error] (compile:compileIncremental) Compilation failed
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore?
```

Probably I will able add this to `src/sbt-test/ensime-sbt` later.

**Comment by strobe** [comment]
looks like it happen after 073a7a3b9a070896e66cbb7da10d9afca01d9693 (sbt-compat)

**Comment by strobe** [comment]
user workaround for this - add to project/plugins.sbt
```scalacOptions in ThisBuild ++= List("-Yresolve-term-conflict:package")```

**Comment by strobe** [comment]
fixed by "sbt-revolver" "0.9.1"
